plugins/modules/zabbix_maintenance.py

```diff
@@ -59,6 +59,12 @@
             - Type of maintenance. With data collection, or without.
         type: bool
         default: 'yes'
+    visible_name:
+        description:
+            - Type of zabbix host name to use for identifying hosts to include in the maintenance.
+            - I(visible_name=yes) to search by visible name,  I(visible_name=no) to search by technical name.
+        type: bool
+        default: 'yes'
```

Comment:
```suggestion
        default: 'yes'
       version_added: ''
```

Sorry, I have missed this.  
Can you please add the version_added of the version to be added?

@D3DeFi 
Am I understanding correctly that the next release version 2.0.0?

Reply:
it's done